src/coreclr/src/vm/interopconverter.cpp

```diff
@@ -466,9 +480,11 @@ IUnknown *GetComIPFromObjectRef(OBJECTREF *poref, REFIID iid, bool throwIfNoComI
     {
         SafeComHolder<IUnknown> pvObj;
         hr = pUnk->QueryInterface(iid, &pvObj);
+        pUnk->Release();
         if (FAILED(hr))
             COMPlusThrowHR(hr);
 
+        pUnk = pvObj.Extract();
```

Comment:
I would avoid reusing the the `pUnk` in this case. Perhaps just `RETURN (pvObj.Extract());`?

Reply:
You do not really need `SafeComHolder` here at all.

Reply:
(`pUnk` does not use it either.)